src/website/index.html

```diff
@@ -7,6 +7,16 @@
     <meta name="description" content="Mineral UI is a design system and React component library for the web that lets you quickly build high-quality, accessible apps. Created by CA Technologies.">
     <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i,800,800i&subset=cyrillic,greek" rel="stylesheet">
     <title>Mineral UI</title>
+    <!-- Global Site Tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-107538931-1"></script>
```

Comment:
Rather than relying on Google Analytics filters to exclude traffic on 0.0.0.0, which is insufficient as local development can be browsed on other IP's as well, let's simply only include the script for production.

The following should do the trick...

_webpack.config.js_
```js
new HtmlWebpackPlugin({
  template: './src/website/index.html',
  isProduction
}),
```

_index.html (supports ejs)_
```js
<% if (htmlWebpackPlugin.options.isProduction) { %>
  ...analytics scripts here
<% } %>
```

Reply:
addressed, and deleted the 0.0.0.0 filter in the ga dashboard.